src/main/studio/kdb/QErrors.java

```diff
@@ -12,38 +12,49 @@ public static String lookup(String s) {
     
 
     static {
-        map.put("access","attempt to read files above directory, run system commands or failed usr/pwd");
+        map.put("access","attempt to read files above directory or run system commands in -u 1 mode, or failed usr/pwd");
         map.put("assign","attempt to assign a value to a reserved word");
-        map.put("conn","too many incoming connections (1022 max)");
+        map.put("badmsg","failure in IPC validation");
+        map.put("badtail","incompelte transaction at end of log file, get good (count;length) with -11!(-2;`:file)");
+        map.put("cast","value not in enumeration");
+        //map.put("close handle.*","handle was closed by the remote while a msg was expected"); //this would require regex matching
+        //map.put("conn","too many incoming connections (1022 max)"); //this is a server-side error, shouldn't happen during a client query
         map.put("domain","out of domain");
-        map.put("glim","`g# limit, kdb+ currently limited to 99 concurrent `g#'s ");
+        map.put("from","badly formed select query");
+        //map.put("glim","`g# limit, kdb+ currently limited to 99 concurrent `g#'s "); //obsolete
```

Comment:
Seems harmless to leave in.  Tail of legacy is long...

Reply:
See my comment on the issue. For this particular one, the limit actually changed between versions, and I wanted to put in the latest value of the limit but turns out there is none.